mithril-aggregator/src/configuration.rs

```diff
@@ -251,7 +251,9 @@ impl Configuration {
             public_server_url: None,
             run_interval: 5000,
             db_directory: PathBuf::new(),
-            snapshot_directory: PathBuf::new(),
+            // Note: this is a band-aid solution to avoid IO operations in the `mithril-aggregator`
+            // crate directory. This should be fixed in the future.
```

Comment:
Maybe you can add few details about the problems not solved (e.g. collisions)?

Reply:
done